doc/source/io.rst

```diff
@@ -435,11 +435,65 @@ individual columns:
     df = pd.read_csv(StringIO(data), dtype={'b': object, 'c': np.float64})
     df.dtypes
 
+Fortunately, ``pandas`` offers more than one way to ensure that your column(s)
+contain only one ``dtype``. For instance, you can use the ``converters`` argument
+of :func:`~pandas.read_csv`:
+
```

Comment:
there is also another section in ``basics.rst`` which has the sub-heading ``Object conversions``, pls add a refernce to there (you will need to update basics.rst with a ``.. _basics.object_conversion`` or something to make the other side of the pointer.